democritus_dates/dates_and_times.py

```diff
@@ -459,3 +459,18 @@ def time_waste(n=3):
     time.sleep(n)
     message = f'I just wasted {n} seconds of your life.'
     print(message)
+
+def time_as_float(time_string: str) -> float:
+    """converts a given HH:MM time string to float"""
+    try:
+        hours, mintues = list(map(int, time_string.split(":"))) #parse given time string
+    except ValueError:
+        message = f"Invalid time string, ensure that the argument is in HH:MM format. Provided value: {time_string}"
+        raise ValueError(message)
```

Comment:
I like how you define the error message in a seperate variable. Some people don't prefer this, but I like it because it makes it easier to change how the error message is used.